hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/CreateTableTypeHelper.java

```diff
@@ -0,0 +1,247 @@
+package com.splicemachine.derby.impl.sql.execute.operations;
+
+import org.junit.Assert;
+
+import java.sql.Clob;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Locale;
+
+/// a helper class to define column types, and then create external tables and insert data into them
+/// to make writing tests for all column types easier.
+
+public class CreateTableTypeHelper {
+    /// @param types: an array of Types that should be used
```

Comment:
Ditto here and all other methods

Reply:
fixed in https://github.com/splicemachine/spliceengine/pull/3811